docs/control-structures.rst

```diff
@@ -900,3 +913,4 @@ in scope in the block that follows.
     The caller always retains at least 1/64th of the gas in a call and thus
     even if the called contract goes out of gas, the caller still
     has some gas left.
+    
```

Comment:
[`chk_coding_style` is failing here due to trailing whitespace](https://app.circleci.com/pipelines/github/ethereum/solidity/37019/workflows/87b2195e-5c4b-4670-9e37-7f958e47c9ab/jobs/1693393). Is the empty line even necessary here?

Reply:
No, feel free to remove it.